build.xml

```diff
@@ -159,6 +159,16 @@
     </target>
 
     <!-- Public API -->
+    <target name="spring4-soffit" description="Rebuild uPortal-soffit packages with Spring 4">
+        <property name="spring3version" value="3.2.9.RELEASE" />
+        <property name="spring4version" value="4.2.6.RELEASE" />
+
+        <replace file="gradle.properties" token="${spring3version}" value="${spring4version}"/>
```

Comment:
I think this comment and the one above are better solved by upgrading the version of Spring in uPortal to version 4. We only have a couple of days left of Spring 3.2.x support: https://spring.io/blog/2015/06/15/spring-framework-3-2-x-eol-on-dec-31-2016 I think leaving this how it is for right now is a good enough band-aid until Spring 4 is a thing for us.